src/routes/api.php

```diff
@@ -32,10 +32,22 @@
         Route::delete('presentations/{presentation}/users/{user}', 'API\PresentationUserController@unsubscribe')->name('presentations.users.unsubscribe');
         // manage user rights on presentations
         Route::put('presentations/{presentation}/users/{user}', 'API\PresentationUserController@changeRole')->name('presentations.change_role');
+
         // poll management
         Route::apiResource('presentations.polls', 'API\PollController')->shallow();
-        Route::put('polls/{poll}/publish', 'API\PollController@publish')->name('polls.publish');
-        Route::get('polls/{poll}/results', 'API\PollController@results')->name('polls.results');
+        Route::middleware(['poll.perms.presenter'])->group(function () {
+            Route::put('presentations/{presentation}/polls', 'API\PollController@store')->name('presentations.polls.store');
+            Route::put('polls/{poll}', 'API\PollController@update')->name('polls.update');
+            Route::get('polls/{poll}/publish', 'API\PollController@publish')->name('polls.publish');
```

Comment:
the verb for this route should be `put` maybe not your fault but if you could change it, would be nice

Reply:
Fixed in 33abdbe0297f431f385c0fcbcb5e0c9145ecf785